cpp/tests/binaryop/binop-compiled-fixed_point-test.cpp

```diff
@@ -0,0 +1,518 @@
+/*
+ * Copyright (c) 2021, NVIDIA CORPORATION.
+ *
+ * Copyright 2018-2019 BlazingDB, Inc.
+ *     Copyright 2018 Christian Noboa Mardini <christian@blazingdb.com>
```

Comment:
I think this copyright is propagating unnecessarily. Blazing didn't write the fixed-point binops tests, I don't think.